app/main.py

```
@@ -1,12 +1,42 @@
 from fastapi import FastAPI
 
 from app.api import router as v1_router
+from fastapi import FastAPI, HTTPException
```

Ruff (0.12.2)

Redefinition of unused `FastAPI` from line 1

Remove definition: `FastAPI`

(F811)